package.json

```diff
@@ -58,7 +57,7 @@
     "stress:test": "node scripts/stress-test/knex-stress-test.js | grep -A 5 -B 60 -- '- STATS '",
     "stress:destroy": "docker-compose -f scripts/stress-test/docker-compose.yml stop",
     "prepare": "husky install && npm run clean && npm run build",
-    "prepublishOnly": "npm run format:check-difference && npm run lint && npm run clean && npm run build"
+    "prepublishOnly": "CI=true npm run lint:everything && npm run clean && npm run build"
```

Comment:
this will fail on Windows.
also why is this CI? it's run locally

Reply:
I'll split it out so it's clearer.

Reply:
Done.